app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/commontask/Structs.kt

```diff
@@ -62,6 +60,9 @@ class EditActions(
     val editEpicColor: EditSimple<String> = EditSimple(),
     val deleteTask: () -> Unit = {},
     val promoteTask: () -> Unit = {},
+    val assigneeOrWatchToMe: EditSimple<Boolean> = EditSimple(),
```

Comment:
Separate action for assign / watch and simple value instead of lambda `check..`. (as I said above)